cdi-unit/pom.xml

```diff
@@ -35,27 +35,24 @@
 		<dependency>
 			<groupId>org.jboss.weld.se</groupId>
 			<artifactId>weld-se-core</artifactId>
-			<scope>compile</scope>
-			<version>2.3.4.Final</version>
+			<version>2.4.2.SP1</version>
+			<scope>provided</scope>
```

Comment:
This is basically fixing a scoping mistake, is that right? If another project uses cdi-unit with test scope, it won't make any difference, but if it uses it with compile scope (for some reason), this will keep weld-se-core off the classpath, which is probably a good thing.